### ISSUE TITLE
expected performance on `train-val` set

### ISSUE THREAD

**Issue by ZENGXH** [imatge-upc/rvos#6]
Hi, 

Thanks so much for releasing the code and the splitting of the training set. Can I ask if you have test the performance of `eval_provious_mask` + the `youtubevos-one-shot` model you released + validation set in `train-val-meta.json`? I just want to double check if my testing is bug free.  

Thanks!

**Comment by carlesventura** [comment]
Sorry but on the validation set (train-val) we only measured the loss, which is based on the Jaccard, but not the official evaluation measures. All models have been evaluated using the official evaluation server from either DAVIS or YouTube-VOS. We have not implemented the evaluation scripts.
